File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int n= nums.length;
        int windex=0;
        for(int i=0; i<n; i++ ){
            if(nums[i]!=0){
                nums[windex]=nums[i];
                windex+=1;

            }
        }
        for(int j=windex; j<n; j++){
            nums[j]=0;
        }
        
        
    }
}

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        insert=0
        for i in range(len(nums)):
            if nums[i]!=0:
                nums[i] , nums[insert]= nums[insert], nums[i]
                insert+=1
        return insert

        """
        Do not return anything, modify nums in-place instead.
        """
        